ccmlib/cluster.py

```diff
@@ -429,6 +429,17 @@ def start(self, no_wait=False, verbose=False, wait_for_binary_proto=False, wait_
                     mark = node.mark_log()
 
                 p = node.start(update_pid=False, jvm_args=jvm_args, profile_options=profile_options, verbose=verbose, quiet_start=quiet_start)
+                # Adding multiple nodes to the cluster in parallel is not
+                # supported by Scylla, at least for now. Scylla is becoming
+                # more strict to check if there are other pending node
+                # operations when a node bootstraps. It fails the bootstrap if
+                # there are pending node operations. To workaround the existing
+                # dtests that adding multiple in parallel, add delay to allow
+                # the second bootstrap node to detect the first bootstrap is in
+                # process and wait until the first node has finished. Starting
+                # nodes at the exact same time makes it very hard to detect
+                # other pending operations in the cluster.
+                time.sleep(2)
```

Comment:
@asias is there an actual problem you encountered or is it just a theoretical concern?

This change is futile since `Cluster.start` is overridden by [ScyllaCluster.start](https://github.com/scylladb/scylla-ccm/blob/a8429536a4ba43200dabd836c9df88c396931937/ccmlib/scylla_cluster.py#L143) that calls [start_nodes](https://github.com/scylladb/scylla-ccm/blob/a8429536a4ba43200dabd836c9df88c396931937/ccmlib/scylla_cluster.py#L85), and the latter does wait on the previous node until it starts listening to CQL, before starting a new node.

Reply:
Ah, I was using a very old ccmlib. I saw that node2 and node3 were started at the same time. After upgrading to latest ccmlib, it worked without this patch. 

Btw, I think we really should move ccmlib under the scylla-dtest repo.